psalm.xml

```diff
@@ -4,7 +4,8 @@
        xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
        errorLevel="1"
        cacheDirectory="./build/cache/psalm"
-       errorBaseline="psalm-baseline.xml">
+       errorBaseline="psalm-baseline.xml"
+       phpVersion="8.1">
```

Comment:
Psalm infers the target PHP version from composer.json by default. Enum classes add in brick/math v0.12 are ignored in PHP 8.0, so fixed to PHP 8.1 as the target to avoid it.